deploy/charts/kubernetes-cluster-prep/templates/clusterrole.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.authnK8s.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.authnK8s.rbac.clusterRole }}
+  labels:
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ .Release.Name }}
```

Comment:
I *think* this should be a more generic, fixed name, e.g.:
```
    app.kubernetes.io/name: authn-k8s-clusterrole
```
based on this reference: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels

Apologies, I think the issue and the design spec were not quite consistent with Kubernetes guidelines.